### PR TITLE
Droth 408 carriageway number to road link attributes

### PR DESCRIPTION
**Pull request by halied**

### CODE REVIEW

**Comment by tvenhola** [comment]
ok

File: UI/src/model/SelectedLinkProperty.js [hunk 1]
```diff
@@ -39,13 +62,23 @@
         var latestModified = dateutil.extractLatestModifications(selectedData);
         var municipalityCodes = {municipalityCode: extractUniqueValues(selectedData, 'municipalityCode')};
         var verticalLevels = {verticalLevel: extractUniqueValues(selectedData, 'verticalLevel')};
-        var roadPartNumbers = {roadPartNumber: extractUniqueValues(selectedData, 'roadPartNumber')};
+        var roadNumbers = extractUniqueValues(selectedData, 'roadNumber');
+        var roadPartNumbers = {roadPartNumber: null};
+        var startAddrMValue = {startAddrMValue: null};
+        var endAddrMValue = {startAddrMValue: null};
```
Comment (marquesfm) [inline UI/src/model/SelectedLinkProperty.js]: Instead "startAddrMValue: null" should be "endAddrMValue: null"

**Comment by joanacoelho** [comment]
ok

**Comment by marquesfm** [comment]
Except what has been commented, everything is ok

**Comment by MAC-CGI** [comment]
Ok, for me

**Comment by leirasf** [comment]
Apart from @marquesfm 's comment this is ok to me.

File: UI/src/model/SelectedLinkProperty.js (hunk 1 above)
Comment (halied) [inline UI/src/model/SelectedLinkProperty.js]: Fixed

**Pull request merged**